draft-ietf-quic-http.md

```diff
@@ -667,30 +667,35 @@ this limit are not guaranteed to be accepted.
 
 ### Request Cancellation and Rejection {#request-cancellation}
 
-Clients can cancel requests by resetting and aborting the request stream with an
-error code of H3_REQUEST_CANCELLED ({{http-error-codes}}).  When the client
-aborts reading a response, it indicates that this response is no longer of
-interest. Implementations SHOULD cancel requests by abruptly terminating any
-directions of a stream that are still open.
-
-When the server rejects a request without performing any application processing,
-it SHOULD abort its response stream with the error code H3_REQUEST_REJECTED.
-In this context, "processed" means that some data from the stream was passed to
-some higher layer of software that might have taken some action as a result. The
-client can treat requests rejected by the server as though they had never been
-sent at all, thereby allowing them to be retried later on a new connection.
-Servers MUST NOT use the H3_REQUEST_REJECTED error code for requests which
-were partially or fully processed.  When a server abandons a response after
-partial processing, it SHOULD abort its response stream with the error code
-H3_REQUEST_CANCELLED.
-
-When a client resets a request with the error code H3_REQUEST_CANCELLED, a
-server MAY abruptly terminate the response using the error code
-H3_REQUEST_REJECTED if no processing was performed.  Clients MUST NOT use the
-H3_REQUEST_REJECTED error code, except when a server has requested closure of
-the request stream with this error code.
-
-If a stream is cancelled after receiving a complete response, the client MAY
+Once a request has been sent, it MAY be cancelled by either endpoint.  Clients
```

Comment:
Does the request body (if exists) need to be sent completely before the client can cancel the request?  Do the request headers need to be sent completely?  Or may the stream be cancelled if only a the very first byte of the HEADERS frame has been delievered in a QUIC STREAM frame?

Would "Once a request stream is open, the request MAY be cancelled" be better?

Reply:
Technically, the stream needn't have any bytes sent.